packages/web/src/scss/settings/_utilities.scss

```diff
@@ -2,6 +2,41 @@
 @use '@tokens' as tokens;
 
 $utilities: (
```

Comment:
I would add border-style utilities with some basic options; we can add more later. 

Also, I am considering renaming and distinguishing the border width, colour and style utilities. Because if we introduce style with the same class as now, we will get border-none (border-style: none), which might be confusing, because we modify the style only.

If we would rename borderWidth utilities to `border-width-<number>`. And have `border-color-basic` `border-style-solid`. 

I feel like this might be better, more readable and so on. However tailwind uses your approach 🤷  I will leave it up to you.

Reply:
Utility class for border-style added. The rest is up for discussion.